src/mono/bclwipe/mscorlib.txt

```diff
@@ -0,0 +1 @@
+# Nothing to do here yet
```

Comment:
Pretty sure we'll add something here soon. For now I just wanted this file to verify we could process multiple spec files.